src/style.css

```diff
@@ -0,0 +1,84 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  padding-top: 12px;
+}
+
+.form {
+  margin-bottom: 20px;
+  position: relative;
+}
+
+.search {
+  font-family: "Avenir", sans-serif;
```

Comment:
Usually we set font related styles to `html` tag, then almost every element will inherit it (if not, you can set `inherit` yourself)